README.md

```diff
@@ -73,12 +59,27 @@ module.**
 * This module does not come with an entity view page for API products at this moment. (See
```

Comment:
Is this still true? I thought a template was created so some edge entities could be rendered? If so, we should update https://github.com/apigee/apigee-edge-drupal/issues/72 and https://github.com/apigee/apigee-edge-drupal/pull/166

Reply:
Actually, this has been resolved by #172.